tests/test_init.py

```
@@ -1674,9 +1676,10 @@ async def test_async_update_listener_device_removal_param(
     make_config_entry: Callable[..., MockConfigEntry],
     dt_enabled: Any,
     via_device_id: Any,
-    expect_removed: Any,
+    tracker_device_id: Any,
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `tracker_device_id`

(ANN401)